app/scripts/models/reliers/relier.js

```diff
@@ -21,7 +21,15 @@ define([
   Constants) {
   'use strict';
 
-  var RELIER_FIELDS_IN_RESUME_TOKEN = ['campaign', 'entrypoint'];
+  var RELIER_FIELDS_IN_RESUME_TOKEN = [
+    'utmTerm',
+    'utmSource',
+    'utmMedium',
+    'utmContent',
+    'utmCampaign',
+    'campaign',
```

Comment:
Huh, adding UTM parameters here doesn't seem right and breaks encapsulation. I thought an OAuth relier should be created in app-start.js when verifying for an OAuth service, and if so, this would not be needed.